src/Congo.RazorPages/Extensions/ServicesExtensions.cs

```diff
@@ -11,12 +14,15 @@ public static void AddAppServices(this IServiceCollection services)
             services.AddScoped<IProductsService, ProductsService>();
         }
 
-        public static void AddHttpClients(this IServiceCollection services)
+        public static void AddHttpClients(this IServiceCollection services, IConfiguration config)
         {
-            services.AddHttpClient(nameof(Congo), config =>
-            {
-                config.BaseAddress = new Uri("https://localhost:44388/api/");
-            });
+            var congoApiUri = new Uri(config["CongoApi"]);
+
+            services.AddRefitClient<ICongoUserClient>()
+                .ConfigureHttpClient(x => x.BaseAddress = congoApiUri);
+
+            services.AddRefitClient<ICongoSellerClient>()
```

Comment:
Client site doesn't need access to seller api

Reply:
Good catch!
Should i also remove the ICongoUserClient from blazor?

Reply:
Eh, I think we can stick with the YAGNI (you ain’t gonna need it) principle and add it back later, if there’s ever a need